test/gtest/common/mem_buffer.cc

```diff
@@ -46,7 +46,7 @@
 bool mem_buffer::is_cuda_supported()
 {
 #if HAVE_CUDA
-    int num_gpus;
+    int num_gpus        = 0;
```

Comment:
same for ROCM?

Reply:
it seems this is CUDA specific issue, at least I have never seen it for ROCM